src/librustc_middle/mir/interpret/error.rs

```diff
@@ -462,9 +462,9 @@ impl fmt::Debug for UnsupportedOpInfo {
         match self {
             Unsupported(ref msg) => write!(f, "{}", msg),
             ReadForeignStatic(did) => {
-                write!(f, "tried to read from foreign (extern) static {:?}", did)
+                write!(f, "cannot read from foreign (extern) static {:?}", did)
             }
-            NoMirFor(did) => write!(f, "could not load MIR for {:?}", did),
+            NoMirFor(did) => write!(f, "cannot load MIR for {:?}", did),
```

Comment:
while it's true that we can't, maybe it would be more helpful to the user if we told them that `"no MIR exists for {:?}"` or `" there is no MIR Body associated with {:?}"`

Reply:
Since rlibs can have partial MIR I think we should not say "there is no MIR", just that we cannot get it... see my latest commit.

Reply:
Oh right, good point